### DISCUSSION TITLE
Open and Close Model Button: Can it toggle on and off

### DISCUSSION THREAD

**Discussion by databayes** [TabularEditor/TabularEditor3#1308] [Ideas]
### Is your feature request related to a problem? Please describe

I frequently open and close the model when I am working with different objects.  Could this button be a toggle?
![delete](https://github.com/TabularEditor/TabularEditor3/assets/87519321/8b615c60-8bab-4885-ac35-600805d7f9f4)

### Describe the solution you'd like

To be able to toggle on and off for open and close model

### Goals

1. Be more efficient
2.

**Comment by otykier** [comment]
Duplicate of #1307
